stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -72,17 +73,20 @@ public static void main(String[] args) {
   public String getBenchDescription() {
     return "This stress bench is designed for testing the reading throughput of Fuse-based "
```

Comment:
Try follow the format in https://github.com/Alluxio/alluxio/pull/13936/files to have easier to understand bench description